### PR TITLE
PLAT-794  | Allow {{currentUserName}} in dashboard news message

### PR DESCRIPTION
**Pull request by rogatty**
https://bethink.atlassian.net/browse/PLAT-794

### CODE REVIEW

File: resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue [hunk 1]
```diff
@@ -53,15 +70,21 @@
...
+				return this.isEdit ? 'put' : 'post';
 			},
 			isEdit() {
 				return this.id !== 'new';
-			}
+			},
+			messageArguments() {
+				return {
+					currentUserName: this.currentUserName
+				};
```
Comment (rogatty) [inline resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue]: We'll need to maintain this list of arguments in two places. Would it be better to use a mixin?

File: resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue [hunk 2]
```diff
@@ -9,26 +9,43 @@
 				@submitSuccess="onSubmitSucess"
 				@change="onChange"
 		>
-			<wnl-text name="slug">Slug</wnl-text>
+			<wnl-text name="slug">Tytuł</wnl-text>
 			<wnl-textarea name="message">Treść</wnl-textarea>
+			<p>Możesz użyć następujących parametrów:</p>
+			<ul class="message-help">
+				<li v-for="argumentKey in Object.keys(messageArguments)">
```
Comment (kvas-damian) [inline resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue]: Please set `:key` attribute to make Vue life easier.

Comment (bognix) [inline resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue]: v-for="(value, key) in messageArguments"

File: resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue [hunk 3]
```diff
@@ -9,26 +9,43 @@
...
 </template>
 
 <style lang="sass" scoped>
 	.notification
 		max-width: 900px
+
+	.message-help
+		list-style: disc
+		margin-bottom: 10px
+		/deep/ li
```
Comment (kvas-damian) [inline resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue]: maybe it's better to style based on classes?

File: resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue (hunk 2 above)
Comment (rogatty) [reply resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue]: Based on https://vuejs.org/v2/guide/list.html#key, I think it makes sense only when iterating over an array of objects.

File: resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue (hunk 1 above)
Comment (rogatty) [reply resources/assets/js/admin/components/dashboardNews/DashboardNewsEdit.vue]: @bognix `named exports are not supported in *.vue files.`

**Pull request merged**